### ISSUE TITLE
fix: add required changes to docusaurus present in  /docs

### ISSUE THREAD

**Issue by ashwinijha6** [anitab-org/open-source-programs-web#115]
### Is your feature request related to a problem? Please describe.
Certain changes are required in docusaurus present in the /docs directory
- Add the command `npm run serve` wherever required one can refer [this](https://github.com/anitab-org/open-source-programs-backend/blob/develop/osp-backend-docusaurus/README.md).
- The .features & .featureImage is not required, should be removed from [here](https://github.com/anitab-org/open-source-programs-web/blob/develop/docs/src/pages/styles.module.css).
- As the open source programs backend docusaurus has been published the link should be changed in [docusaurus.config.js](https://github.com/anitab-org/open-source-programs-web/blob/develop/docs/docusaurus.config.js)

### Describe the solution you'd like
- `npm run serve` will help checking with the local build during change
-  The 2 features mentioned above are not required
-  Correct link should be mentioned

### Additional context
![npm](https://user-images.githubusercontent.com/45794450/110295983-ca34e780-8017-11eb-9fc2-7ae99f81562e.JPG)
![feature_not_required](https://user-images.githubusercontent.com/45794450/110296012-d28d2280-8017-11eb-962e-c127ca7d56a8.JPG)
![doc_backend](https://user-images.githubusercontent.com/45794450/110296041-dd47b780-8017-11eb-8714-afab9997388b.JPG)

**Comment by ashwinijha6** [comment]
@codesankalp & @keshakaneria could u'll pls check this issue & add necessary labels . Also if it is valid I would like to work on it.

**Comment by keshakaneria** [comment]
In the footer, the 'Backend docs' needs to have the same link which is there currently, i guess. But yes, the `npm run serve` and unwanted .css can be removed from the file.

**Comment by ashwinijha6** [comment]
> In the footer, the 'Backend docs' needs to have the same link which is there currently, i guess. But yes, the `npm run serve` and unwanted .css can be removed from the file.

No @keshakaneria we have migrated the docs to docusaurus for backend & web separately but the web was merged before the backend that's why the link needs to be changed in the **Backend docs** mentioned in the footer to [this](https://osp-backend-docs.surge.sh/) currently it is the wiki link.

**Comment by keshakaneria** [comment]
> > In the footer, the 'Backend docs' needs to have the same link which is there currently, i guess. But yes, the `npm run serve` and unwanted .css can be removed from the file.
> 
> No @keshakaneria we have migrated the docs to docusaurus for backend & web separately but the web was merged before the backend that's why the link needs to be changed in the **Backend docs** mentioned in the footer to [this](https://osp-backend-docs.surge.sh/) currently it is the wiki link.

Oh yes I'm sorry I forgot we separated them 🤦‍♀️ Yes the link need to be changed! You can go ahead solving this issue 👍😄

**Comment by ashwinijha6** [comment]
Hey @keshakaneria should I also change the repo name to anitab-forms-web wherever required or we can keep as another issue like may be a first timers only. What say?

**Comment by keshakaneria** [comment]
> Hey @keshakaneria should I also change the repo name to anitab-forms-web wherever required or we can keep as another issue like may be a first timers only. What say?

The issue is already created and a [PR](https://github.com/anitab-org/anitab-forms-web/pull/121) is also under review. You will have to change the name according to it otherwise while merging it will show conflicts.

**Comment by ashwinijha6** [comment]
> > Hey @keshakaneria should I also change the repo name to anitab-forms-web wherever required or we can keep as another issue like may be a first timers only. What say?
> 
> The issue is already created and a [PR](https://github.com/anitab-org/anitab-forms-web/pull/121) is also under review. You will have to change the name according to it otherwise while merging it will show conflicts.

Yes I found that @keshakaneria I have not changed the repo name here in this PR. I'll let that be done in the that PR only will make review wherever necessary 👍

**Comment by isabelcosta** [comment]
@nlok5923 would you like to contribute to this one?
cc @codesankalp

**Comment by nlok5923** [comment]
Thanks @isabelcosta for picking issue for me i would like to give it a try.
